src/Service/PackageSynchronizer/ComposerPackageSynchronizer.php

```diff
@@ -50,6 +54,15 @@ public function synchronize(Package $package): void
             $json = ['packages' => []];
             $packages = $repository->getPackages();
 
+            // @todo Could we use \Composer\Semver\Semver::sort() instead?
+            usort($packages, static function (PackageInterface $a, PackageInterface $b): int {
```

Comment:
I need to talk it over with @karniv00l, but initially yes :wink:

Reply:
@giggsey What's your intention here? You already did more than `sort()` provides

Reply:
Reply via email didn't work...

The usort fixes a bug with the initial $latest being ahead of a valid stable release.

The Composer sort function deals with Dev releases, which becomes more important when you start restricting the number of releases.

Reply:
I just experimented with this and the only difference is that it normalises the versions first. But the versions are already normalised in the $packages list.

I'll remove the todo.